content/en/watchdog/insights.md

```diff
@@ -284,24 +287,34 @@ For Kubernetes Explorer, the Watchdog Insight carousel reflects [all the Kuberne
 
 [1]: https://app.datadoghq.com/orchestration/overview/pod
 {{% /tab %}}
+
+{{% tab "Cloud Cost Management" %}}
+
+For Cloud Cost, the Watchdog Insight carousel analyzes outliers to automatically [reflect arising trends][1] that you can investigate to nip potential cost explosions in the bud.
```

Comment:
```suggestion
For Cloud Cost, the Watchdog Insight carousel analyzes outliers to automatically [reflect arising trends][1] that you can investigate to prevent potential cost explosions.
```